quarkus/admin/distribution/LICENSE

```diff
@@ -207,1314 +207,898 @@ This binary artifact bundles the following projects:
-License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Group: io.perfmark Name: perfmark-api Version: 0.27.0
+Project URL: https://github.com/perfmark/perfmark
+License: Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: io.swagger Name: swagger-core Version: 1.6.15
-Project URL (from POM): https://github.com/swagger-api/swagger-core
-License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Group: io.projectreactor.netty Name: reactor-netty-core Version: 1.2.5
+Group: io.projectreactor.netty Name: reactor-netty-http Version: 1.2.5
+Project URL: https://github.com/reactor/reactor-netty
+License: Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: io.swagger Name: swagger-jaxrs Version: 1.6.15
-Project URL (from POM): https://github.com/swagger-api/swagger-core
-License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Group: io.projectreactor Name: reactor-core Version: 3.7.5
+Project URL: https://github.com/reactor/reactor-core
+License: Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
+Group: io.swagger Name: swagger-annotations Version: 1.6.15
+Group: io.swagger Name: swagger-core Version: 1.6.15
+Group: io.swagger Name: swagger-jaxrs Version: 1.6.15
 Group: io.swagger Name: swagger-models Version: 1.6.15
-Project URL (from POM): https://github.com/swagger-api/swagger-core
-License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Project URL: https://github.com/swagger-api/swagger-core
+License: Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
 Group: jakarta.activation Name: jakarta.activation-api Version: 2.1.3
-Project URL (from POM): https://github.com/jakartaee/jaf-api
-License (from POM): EDL 1.0 - http://www.eclipse.org/org/documents/edl-v10.php
+Project URL: https://github.com/jakartaee/jaf-api
+License: EDL 1.0 - http://www.eclipse.org/org/documents/edl-v10.php
 
 --------------------------------------------------------------------------------
 
 Group: jakarta.annotation Name: jakarta.annotation-api Version: 3.0.0
-Project URL (from POM): https://projects.eclipse.org/projects/ee4j.ca
-License (from POM): EPL 2.0 - https://www.eclipse.org/legal/epl-2.0
-License (from POM): GPL2 w/ CPE - https://www.gnu.org/software/classpath/license.html
+Project URL: https://projects.eclipse.org/projects/ee4j.ca
+License: EPL 2.0 - https://www.eclipse.org/legal/epl-2.0
+License: GPL2 w/ CPE - https://www.gnu.org/software/classpath/license.html
```

Comment:
Where things are dual licensed (and that's not entirely clear from the above), you should select the license and not list GPL as it is category X. See https://www.apache.org/legal/resolved.html#mutually-exclusive

Reply:
Yes I remember. Let me select the right license. Thanks !

Reply:
Done.